imports/client/components/HuntEditPage.tsx

```diff
@@ -0,0 +1,558 @@
+                <DiscordChannelSelector
+                  id="hunt-form-puzzle-hooks-discord-channel"
+                  guildId={guildId}
+                  disable={disableForm}
+                  value={puzzleHooksDiscordChannel}
+                  onChange={onPuzzleHooksDiscordChannelChanged}
+                />
+                <FormText>
+                  If this field is specified, when a puzzle in this hunt is added or solved, a
+                  message will be sent to the specified channel.
+                </FormText>
+              </Col>
+            </FormGroup>
+
+            <FormGroup as={Row}>
+              <FormLabel column xs={3} htmlFor="hunt-form-firehose-discord-channel">
+                Firehose Discord channel
+              </FormLabel>
+              <Col xs={9}>
+                <DiscordChannelSelector
+                  id="hunt-form-firehose-discord-channel"
+                  guildId={guildId}
+                  disable={disableForm}
+                  value={firehoseDiscordChannel}
+                  onChange={onFirehoseDiscordChannelChanged}
+                />
+                <FormText>
+                  If this field is specified, all chat messages written in puzzles associated with
+                  this hunt will be mirrored to the specified Discord channel.
+                </FormText>
+              </Col>
+            </FormGroup>
+
+            <FormGroup as={Row}>
+              <FormLabel column xs={3} htmlFor="hunt-form-member-discord-role">
+                Discord role for members
+              </FormLabel>
+              <Col xs={9}>
+                <DiscordRoleSelector
+                  id="hunt-form-member-discord-role"
+                  guildId={guildId}
+                  disable={disableForm}
+                  value={memberDiscordRole}
+                  onChange={onMemberDiscordRoleChanged}
+                />
+                <FormText>
+                  If set, then members of the hunt that have linked their Discord profile are added
+                  to the specified Discord role. Note that for continuity, if this setting is
+                  changed, Jolly Roger will not touch the old role (e.g. to remove members)
+                </FormText>
+              </Col>
+            </FormGroup>
+          </>
+        ) : (
+          <Alert variant="info">
+            <FontAwesomeIcon icon={faInfo} fixedWidth />
+            Discord has not been configured, so Discord settings are disabled.
+          </Alert>
+        )}
+
+        <div ref={footer}>
+          {submitState === SubmitState.FAILED && <Alert variant="danger">{errorMessage}</Alert>}
+          {submitState === SubmitState.SUCCESS && (
+            <Alert variant="success" dismissible onClose={onSuccessDismiss}>
+              Hunt information successfully updated
+            </Alert>
+          )}
+
+          <Button variant="secondary" type="button" onClick={goBack}>Go back</Button>
```

Comment:
While the previous modal form needed a way to dismiss it, since getting to this page now involves a navigation, I feel like between the browser back button and the breadcrumbs we don't need an additional way to do this.  This is no longer a modal form; it no longer needs its own "dismiss" button.  Can we take it out?

Reply:
Haha. Honestly, I added the back button because I expected you to want the explicit navigation/nudge for post-save action. I don't mind dropping it